packages/web-pkg/src/composables/actions/useActionsShowDetails.ts

```diff
@@ -0,0 +1,27 @@
+import { computed } from 'vue'
+import { useGettext } from 'vue3-gettext'
+import { eventBus } from 'web-pkg'
+import { SideBarEventTopics } from 'web-pkg/src/composables/sideBar'
+import { Action } from './types'
+
+export const useActionsShowDetails = () => {
+  const { $gettext } = useGettext()
+
+  const actions = computed((): Action[] => [
+    {
+      name: 'show-details',
+      icon: 'information',
+      label: () => $gettext('Details'),
+      handler: () => eventBus.publish(SideBarEventTopics.open),
+      isEnabled: ({ resources }) => {
+        return (resources as unknown[]).length > 0
+      },
+      componentType: 'button',
+      class: 'oc-admin-settings-show-details-trigger'
```

Comment:
```suggestion
      class: 'oc-show-details-trigger'
```

or 

```suggestion
      class: 'oc-generic-show-details-trigger'
```

or something similar? composable is (gladly) generic and not part of the admin settings app anymore.

Same for other action composables in this PR. Can be done in a followup. Also needs adjustment of element selectors in tests.

Reply:
uh, yeah - let's do that in a followup